CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ if( NOT BUILD_RELEASE )
 endif()
 STRING(COMPARE EQUAL ${CMAKE_BUILD_TYPE} "PRODUCTION" BUILD_PRODUCTION)
 
-STRING(COMPARE EQUAL ${CMAKE_BUILD_TYPE} "DEGUG" BUILD_DEBUG)
+STRING(COMPARE EQUAL ${CMAKE_BUILD_TYPE} "DEBUG" BUILD_DEBUG)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
 find_package( Jasper )
```

Comment:
These packages are required. Should they be marked as REQUIRED. See:

https://github.com/NOAA-EMC/NCEPLIBS-g2/pull/1/commits/c86bf9fa9ceb5e8055642c0a6a5c1be752c69b39

Reply:
I am not going to make this change now, because we'd have to do that throughout the entire build system to be consistent. We should keep this in mind and make the change in the future if people feel we should, but then in all places imo.

Reply:
Actually, thinking about how much work it is to create these associated 18 or more PRs each time, this would be a good time to do so. @mark-a-potts @kgerheiser what is your opinion?